### DISCUSSION TITLE
flow : action block

### DISCUSSION THREAD

**Discussion by nervusdm** [directus/directus#14965] [Archive]
Hey !

Sorry for asking this....
I wonder how can I use a flow "action block".
I mean , if I use it, no matter what I try, it will... block.
How can I achieve a action block which "block" only if something is true. (conditionnal or something).
 I  should doing something wrong

**Comment by rijkvanzanten** [comment]
I'm not following.. Are you talking about the _Event Hook_ trigger and asking about the difference of _Filter (Blocking)_ and _Action (Non-Blocking)_?
<img width="640" alt="CleanShot 2022-08-09 at 11 01 44@2x" src="https://user-images.githubusercontent.com/9141017/183685760-cba2aa4f-415b-4676-8e99-9ed53bc0882d.png">

**Reply by nervusdm** [reply]
My bad.

I wonder how can I use filter "blocking", and if my flows is ok, update / create the item in collection.

**Reply by rijkvanzanten** [reply]
@nickrum?

**Reply by azrikahar** [reply]
> I wonder how can I use filter "blocking", and if my flows is ok, update / create the item in collection.

There is an open PR #15283 which will make it possible to reject/block **Filter** event hooks. (once again reiterating that it's only for `Filter (Blocking)`, not for `Action (Non-Blocking)`, in case anyone gets confused by the title of this discussion)
